types/cobe/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for cobe 0.4
+// Project: https://github.com/shuding/cobe
+// Definitions by: Martin Badin <https://github.com/martin-badin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import Phenomenon from 'phenomenon';
+
+export as namespace cobe;
+
+export interface Marker {
+    location: [number, number];
+    size: number;
+}
+
+export interface Options {
+    baseColor?: number[];
+    dark?: number;
+    devicePixelRatio?: number;
+    diffuse?: number;
+    glowColor?: number[];
+    height: number;
+    mapBrightness?: number;
+    mapSamples?: number;
+    markerColor?: number[];
+    markers: Marker[];
+    phi?: number;
+    theta?: number;
+    width: number;
+
+    onRender?(state: Record<string, unknown>): void;
+}
+
+declare function createGlobe(canvas: HTMLCanvasElement, opts: Options): Phenomenon;
+
+export default createGlobe;
```

Comment:
Similarly, I think this should be something like:

```ts

declare namespace cobe {
    // All the stuff that is exported above
}

declare function cobe(canvas: HTMLCanvasElement, opts: Options): Phenomenon;

export = cobe;
```

Reply:
(But, I believe `export as namespace cobe` remains as-is, as the lib's docs imply it can be globally available)

Reply:
thanks @jakebailey. I used your suggestion.